app/main.py

```diff
@@ -1,3 +1,16 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        config = json.load(file)
+
+    fuel_price = config["FUEL_PRICE"]
+    shops = [Shop(**shop) for shop in config["shops"]]
+    customers = [Customer(**customer) for customer in config["customers"]]
```

Comment:
you can create a customer in a loop on line 15 and call .go_shopping() there - thus you will not need to do two iterations